### PR TITLE
Adding `simdutf`

### PR DESCRIPTION
**Pull request by SimeonStoykovQC**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/simdutf/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Copyright 2021 The simdutf authors
```
Comment (xhochy) [inline recipes/simdutf/LICENSE.txt:1]: The license is already part of the tarball and should not be duplicated here.

File: recipes/simdutf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simdutf/simdutf/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e193c4b5912612147c266ea7ebe2108099f3bcab5f0dc34e6289b2a47495ae2f
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/simdutf/meta.yaml:13]: As you don't have a build script for windows, you need to skip it here.
```suggestion
  number: 0
  skip: true  # [win]
```

File: recipes/simdutf/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - test -f $CONDA_PREFIX/include/simdutf.h  # [unix]
+    - test -f $CONDA_PREFIX/lib/libsimdutf$SHLIB_EXT  # [unix]
+
+about:
+  home: https://github.com/simdutf/simdutf
+  summary: 'Unicode at GB/s.'
+  description: 'Unicode routines (UTF8, UTF16, UTF32): billions of characters per second using SSE2, AVX2, NEON, AVX-512.'
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (xhochy) [inline recipes/simdutf/meta.yaml:33]:
```suggestion
  license_file: LICENSE-MIT
```

File: recipes/simdutf/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  doc_url: https://simdutf.github.io/simdutf/
+  dev_url: https://github.com/simdutf/simdutf
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
```
Comment (xhochy) [inline recipes/simdutf/meta.yaml:40]: Please remove these comments.
```suggestion
```

File: recipes/simdutf/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simdutf/simdutf/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e193c4b5912612147c266ea7ebe2108099f3bcab5f0dc34e6289b2a47495ae2f
+
+build:
+  number: 0
+
```
Comment (xhochy) [inline recipes/simdutf/meta.yaml:14]: Shared libraries should come with a run_exports.
```suggestion
  run_exports:
    - {{ pin_subpackage('simdutf', max_pin='x') }}
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/simdutf/bld.bat [hunk 6]
```diff
@@ -0,0 +1,4 @@
+mkdir build
+pushd build
+cmake %CMAKE_ARGS% -GNinja -DSIMDUTF_TOOLS=OFF -DBUILD_SHARED_LIBS=ON ..
```
Comment (xhochy) [inline recipes/simdutf/bld.bat:3]: Please add a check to line 3 and 4 that the commands succeeded. CMD doesn't do this by default.

File: recipes/simdutf/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+source:
+  url: https://github.com/simdutf/simdutf/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e193c4b5912612147c266ea7ebe2108099f3bcab5f0dc34e6289b2a47495ae2f
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('simdutf', max_pin='x') }}
+requirements:
```
Comment (xhochy) [inline recipes/simdutf/meta.yaml:16]:
```suggestion

requirements:
```

File: recipes/simdutf/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    - cmake
+    - ninja
+
+test:
+  commands:
+    - test -f $CONDA_PREFIX/include/simdutf.h  # [unix]
+    - if not exist "%LIBRARY_INC%\simdutf.h" exit 1  # [win]
+
+    - test -f $CONDA_PREFIX/lib/libsimdutf$SHLIB_EXT  # [unix]
+    - if not exist "%LIBRARY_LIB%\simdutf.lib" exit 1  # [win]
```
Comment (xhochy) [inline recipes/simdutf/meta.yaml:29]:
```suggestion
    - if not exist "%LIBRARY_BIN%\simdutf.dll" exit 1  # [win]
    - if not exist "%LIBRARY_LIB%\simdutf.lib" exit 1  # [win]
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/simdutf/meta.yaml (hunk 7 above)
Comment (SimeonStoykovQC) [reply recipes/simdutf/meta.yaml:16]: Can this be added to the linter

**Review by xhochy — APPROVED** [review]

**Pull request merged**